src/Controller/Manage/CustomFieldOptionsController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Eventum (Issue Tracking System) package.
+ *
+ * @copyright (c) Eventum Team
+ * @license GNU General Public License, version 2 or later (GPL-2+)
+ *
+ * For the full copyright and license information,
+ * please see the COPYING and AUTHORS files
+ * that were distributed with this source code.
+ */
+
+namespace Eventum\Controller\Manage;
+
+use Custom_Field;
+use Eventum\Controller\Helper\MessagesHelper;
+use User;
+
+class CustomFieldOptionsController extends ManageBaseController
+{
+    /** @var string */
+    protected $tpl_name = 'manage/custom_field_options.tpl.html';
+
+    /** @var int */
+    protected $min_role = User::ROLE_ADMINISTRATOR;
+
+    /** @var string */
+    private $cat;
+
+    /** @var int */
+    private $fld_id;
+
+    /** @var array */
+    private $field_info;
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure()
+    {
+        $request = $this->getRequest();
+
+        $this->fld_id = $request->request->get('fld_id') ?: $request->query->get('fld_id');
+        $this->field_info = Custom_Field::getDetails($this->fld_id);
+
+        $this->cat = $request->request->get('cat') ?: $request->query->get('cat');
+
+        if (empty($this->field_info)) {
```

Comment:
I've moved this to prepareTemplate as I need it on the view page, not the update action.